test/mango.py

```diff
@@ -32,8 +33,7 @@ def __init__(self, host, port, dbname, auth=None):
         self.port = port
         self.dbname = dbname
         self.sess = requests.session()
-        if auth is not None:
-            self.sess.auth = auth
+        self.sess.auth = ('adm', 'pass')
```

Comment:
That was a surprise (: Would you like to update README as well? Otherwise, it's not oblivious with which credentials start CouchDB to test. Previously, admin party mode was good enough for testing.